### DISCUSSION TITLE
Group chat for package

### DISCUSSION THREAD

**Discussion by EzizAnnagurbanow** [flyerhq/flutter_chat_ui#676] [Q&A] [answered]
Hi can you create group chat for this package?

**Comment by demchenkoalex** [comment, marked as answer]
each message has an `author`. You only supply 1 user to the chat - the one who "logged in". Each other `author` is a member of a group. You can show user names/avatars to differentiate between users, making this a group chat. In `v2` user names/avatars is not yet available.
